components/mpas-ocean/bld/build-namelist

```diff
@@ -707,10 +707,11 @@ add_default($nl, 'config_flux_attenuation_coefficient_runoff');
 # Namelist group: coupling #
 ############################
 
+add_default($nl, 'config_remove_ais_river_runoff');
```

Comment:
This is a non-BFB change for simulations with DIB.  @darincomeau and @cbegeman, do you agree that we no longer want to remove river runoff?  Or should we make this change a separate PR?

Reply:
I think we mostly likely want to make this change, but I'd like to see a 30-year B-case before deciding to make this the default.

Reply:
I think that will be a lot to be able to afford for a SORRM run just to test this config option, but we can discuss it.